fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/exceptionhandlers/ExternalContentAccessExceptionMapper.java

```diff
@@ -45,7 +45,7 @@
 
     @Override
     public Response toResponse(final ExternalContentAccessException exception) {
-        debugException(this, exception, LOGGER);
```

Comment:
This class no longer really "implements" `ExceptionDebugLogging`. Should we systematically refactor the use of `ExceptionDebugLogging`?

It seems like this PR does a partial job of that.

Reply:
That did occur to me. I'm happy to remove the ExceptionDebugLogging from these classes. I decided to leave it so that all of the mappers were the same, and in case someone wanted to use that function in the future, but I have no problem removing it, if that's your vote.